src/Activity.js

```diff
@@ -22,12 +27,12 @@ class Activity {
     return this.getWeek(id, date).reduce((acc, obj) => acc + obj.numSteps, 0);
   }
 
-  getMilesByDay(id, date, userRepo) {
+  getMilesByDay(id, date) {
     let feetWalked = this.getDay(id, date).numSteps * userRepo.getUserData(id).strideLength;
```

Comment:
This is one of the lines where you were getting the reference error. Passing in UserRepo as an argument to this method (and any other methods that require it) may be the easiest fix